fed/mosaic-mapping/src/main/java/org/eclipse/mosaic/fed/mapping/ambassador/spawning/InitializableSpawner.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2020 Fraunhofer FOKUS and others. All rights reserved.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contact: mosaic@fokus.fraunhofer.de
+ */
+
+package org.eclipse.mosaic.fed.mapping.ambassador.spawning;
+
+import org.eclipse.mosaic.fed.mapping.ambassador.SpawningFramework;
+import org.eclipse.mosaic.rti.api.InternalFederateException;
+
+public interface InitializableSpawner {
```

Comment:
Good one, we could also name this class `Spawner`. you decide :)

Reply:
Renamed to `Spawner`. 

I'm not a 100% satisfied with this right now.  The `init(SpawningFramework spawningFramework)` method could be part of `UnitSpawner` if traffic light spawning would be handled by the `init(...)`-method as well. I think in order for that to work all spawners would have to wait upon the `ScenarioTrafficLightRegistration`-interaction. Maybe we can address that in a future issue.

Reply:
Can you leave a TODO on the new Spawner interface for that?

Reply:
Added.